### DISCUSSION TITLE
How to show the exit/close button on the tooltip window's upper-right corner?

### DISCUSSION THREAD

**Discussion by jiyang-sas** [gilbarbara/react-joyride#644] [Q&A] [answered]
I replaced the tooltip component with a customer component. Now I don't see the exit/close button on the upper-right. Is that something I need add myself? thx

**Comment by jiyang-sas** [comment]
I tried to use the floater property :
floaterProps={{ showCloseButton: true }}
and I still don't see the "X" button at the right-top corner.

**Comment by gilbarbara** [comment, marked as answer]
hey @jiyang-sas 
Check the [docs](https://docs.react-joyride.com/custom-components)

you need to render it by yourself and use the `closeProps`.
